DigitalLearningSolutions.Data/DataServices/SelfAssessmentDataService/CandidateAssessmentsDataService.cs

```diff
@@ -0,0 +1,217 @@
+﻿namespace DigitalLearningSolutions.Data.DataServices.SelfAssessmentDataService
+{
+    using System;
+    using System.Collections.Generic;
+    using Dapper;
+    using DigitalLearningSolutions.Data.Models.SelfAssessments;
+    using Microsoft.Extensions.Logging;
+
+    public partial class SelfAssessmentDataService
+    {
+        public IEnumerable<CurrentSelfAssessment> GetSelfAssessmentsForCandidate(int candidateId)
+        {
+            return connection.Query<CurrentSelfAssessment>(
+                @"SELECT
+                        CA.SelfAssessmentID AS Id,
+                        SA.Name,
+                        SA.Description,
+                        SA.IncludesSignposting,
+                        SA.SupervisorResultsReview AS IsSupervisorResultsReviewed,
+                        COALESCE(SA.Vocabulary, 'Capability') AS Vocabulary,

```

Comment:
just curious - never seen COALESCE before, it returns the first non-null value in a list? What is the purpose of putting the string 'Capability' in there?

Reply:
Capability is the fallback value that will be returned if SA.Vocabulary is null.